.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
should we have this file? is this not for local purposes?

Reply:
It is useful for debugging. Everyone uses vscode, so I thought we might as well commit it. Let me know if you disagree.